src/components/pages/AddStaff/AddStaffForm.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import uuid from 'uuid';
+import {
+  Container,
+  Button,
+  Grid,
+  Header,
+  Segment,
+  Form,
+  Message,
+  Checkbox
+} from 'semantic-ui-react';
+
+const AddStaff = props => {
+  const { state, handleSubmit, handleChange, checkboxChange } = props;
+  setTimeout(300, () => (state.message = ''));
+  return (
+    <Container>
+      <Grid style={{ height: '85vh' }} centered>
+        <Grid.Column style={{ maxWidth: 350 }}>
+          {state.message && <Message success>{state.message}</Message>}
+          {state.errors.length && (
+            <Message error>
+              <Message.List>
+                {state.errors.map(error => (
+                  <div key={uuid()}>
+                    <Message.Item>{`${Object.values(error)}`}</Message.Item>
+                  </div>
+                ))}
+              </Message.List>
+            </Message>
+          )}{' '}
+          <Header as='h3' style={{ marginBottom: '30px', textAlign: 'center' }}>
+            Create New Account
+          </Header>
+          <Form onSubmit={handleSubmit}>
+            <Segment width={5}>
+              <Form.Field>
+                <label htmlFor='username'>Username</label>
+                <Form.Input
+                  fluid
+                  id='username'
+                  name='username'
+                  type='text'
+                  value={state.username}
+                  onChange={handleChange}
+                  data-testid='username-field'
+                />
+              </Form.Field>
+              <Form.Field>
+                <label htmlFor='password'>Password</label>
+                <Form.Input
+                  fluid
+                  id='password'
+                  name='password'
+                  type='password'
+                  value={state.password}
+                  onChange={handleChange}
+                  data-testid='password-field'
+                />
+              </Form.Field>
+              <Form.Field>
+                <Checkbox
+                  label='Is Admin?'
```

Comment:
I think this label should be more user friendly and descriptive. The data can be called `isAdmin` on backend/database, but the UI should be more clear on what that checkbox is doing.
Some ideas:
- Check this box to make this user Admin
- If you want this user to have Admin rights, please check this box
- Give this user Admin rights
...